cdc/redo/manager.go

```diff
@@ -404,9 +416,9 @@ func (m *ManagerImpl) prepareForFlush() (tableRtsMap map[model.TableID]model.Ts,
 		unflushed := rts.getUnflushed()
 		flushed := rts.getFlushed()
 		if unflushed > flushed {
-			tableRtsMap[tableID] = unflushed
 			flushed = unflushed
```

Comment:
if flushed <= unflushed , the tableRtsMap[tableID] = nil before ? which will cause some problems ?

Reply:
In the patch, `prepareForFlush` returns all tables' resolved timestamps, even if they are not changed.

Reply:
The reason is LogWriter needs the information to construct a redo meta.